datasets/README.md

```diff
@@ -22,14 +25,23 @@ Thanks to using Hugging Face's `datasets` used under the hood, Flower Datasets i
 * Arrow.
 
 Create **custom partitioning schemes** or choose from the **implemented partitioning schemes**:
+
 * Partitioner (the abstract base class) `Partitioner`
 * IID partitioning `IidPartitioner(num_partitions)`
-* Natural ID partitioner `NaturalIdPartitioner`
+* Dirichlet partitioning `DirichletPartitioner(num_partitions, partition_by, alpha)`
+* InnerDirichlet partitioning `InnerDirichletPartitioner(partition_sizes, partition_by, alpha)`
+* Natural ID partitioner `NaturalIdPartitioner(partition_by)`
 * Size partitioner (the abstract base class for the partitioners dictating the division based the number of samples) `SizePartitioner`
-* Linear partitioner `LinearPartitioner`
-* Square partitioner `SquarePartitioner`
-* Exponential partitioner `ExponentialPartitioner`
-* more to come in future releases.
+* Linear partitioner `LinearPartitioner(num_partitions)`
+* Square partitioner `SquarePartitioner(num_partitions)`
+* Exponential partitioner `ExponentialPartitioner(num_partitions)`
+* more to come in the future releases (contributions are welcome).
+<p align="center">
+  <img src="./doc/source/_static/readme/comparison_of_partitioning_schemes.png" alt="Comparison of partitioning schemes."/>
+  <br>
+  <em>Comparison of Partitioning Schemes on CIFAR10</em>
+</p>
+PS: This plot was generated using a library function (see flwr_datasets.visualization package for more).
```

Comment:
```suggestion
PS: This plot was generated using a library function (see [flwr_datasets.visualization](https://flower.ai/docs/datasets/ref-api/flwr_datasets.visualization.html) package for more).
```

Reply:
or maybe we want to update this later after the visualization tutorial page is up?